contracts/governance/Staking/Staking.sol

```diff
@@ -373,6 +472,66 @@ contract Staking is
         emit StakingWithdrawn(msg.sender, amount, until, receiver, isGovernance);
     }
 
+    /**
+     * @notice Send user' staked tokens to a receiver.
+     * This function is dedicated only for direct withdrawal from staking contract.
+     * Currently only being used by governanceDirectWithdrawVesting()
+     *
+     * @param amount The number of tokens to withdraw.
+     * @param until The date until which the tokens were staked.
+     * @param receiver The receiver of the tokens. If not specified, send to the msg.sender.
+     * @param vestingConfig The vesting config.
+     * @dev VestingConfig struct intended to avoid stack too deep issue, and it contains this properties:
+        address vestingAddress; // vesting contract address
+        uint256 startDate; //start date of vesting
+        uint256 endDate; // end date of vesting
+        uint256 cliff; // after this time period the tokens begin to unlock
+        uint256 duration; // after this period all the tokens will be unlocked
+        address tokenOwner; // owner of the vested tokens
+     * */
+    function _withdrawDirect(
```

Comment:
_withdrawFromTeamVesting

Reply:
updated in this [commit](https://github.com/DistributedCollective/Sovryn-smart-contracts/pull/447/commits/19ef0dedeffdf80545269ffafe2f1af684a25a06)